lib/options_tls_go1_12.go

```diff
@@ -0,0 +1,35 @@
+// +build !go1.13
+
+/*
+ *
+ * k6 - a next-generation load testing tool
+ * Copyright (C) 2019 Load Impact
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package lib
+
+import "os"
+
+// TLS 1.3 is available only on an opt-in basis in pre go1.13.
+// To enable it, set the GODEBUG environment variable to contains "tls13=1".
+//
+// Note that this function must be called before any usage of TLS.
+func (o Options) enableTLS13() {
```

Comment:
Ah, wait, nevermind! :blush: The way you've implemented it is actually slightly better, since it will enable TLS 1.3 only if users explicitly request it, which with Go 1.12 is probably the better approach - that way we won't modify the global environment variables if users don't care about TLS 1.3.

Unfortunately, that approach has another minor issue - the `Options.Apply()` method is called multiple times per k6 run, so `enableTLS13()` might also be called multiple times. Can you stick a `sync.Once()` somewhere in the environment variable modification code, so we only do it once?

Reply:
@na-- I also think about it, but if we put it in init, then TLS 1.3 will always be used, even if we specify max version `tls1.2` in script.